Source/x11/XGServerWindow.m

```diff
@@ -4177,29 +4177,32 @@ - (void) standardcursor: (int)style : (void **)cid
     case GSIBeamCursor:
       cursor = XCreateFontCursor(dpy, XC_xterm);
       break;
-    case GSCrosshairCursor:
-      cursor = XCreateFontCursor(dpy, XC_crosshair);
+    case GSClosedHandCursor:
+      cursor = XCreateFontCursor(dpy, XC_fleur);
       break;
-    case GSDisappearingItemCursor:
-      cursor = XCreateFontCursor(dpy, XC_shuttle);
+    case GSOpenHandCursor:
+      cursor = XCreateFontCursor(dpy, XC_hand1);
```

Comment:
For me this is rather a pointing hand.

Reply:
Adwaita theme (standard de-facto) has an open hand. Look for the table I've created some time ago https://github.com/trunkmaster/nextspace/wiki/Mouse-Cursors